docs/src/domains/quick-start.md

```diff
@@ -2,38 +2,68 @@
 title: "Quick Start"
 weight: -70
 description: |
-  By default, a Platform.sh app is available at a domain generated by Platform.sh. The following resources will help you take your application live on your custom domain.
+  By default, a Platform.sh app is available at a domain generated by Platform.sh. The following resources help you take your application live on your custom domain.
 ---
 {{% description %}}
 
-{{< note theme="warning">}}
+This page gives a basic configuration example.
+For more details, see the [Step by step guide](../domains/steps/_index.md).
 
-Adding a custom domain means your existing automatically generated URLs stop working.
+## Before you begin
 
-Don't add a custom domain to your project until you're fully ready to change your DNS settings.
-Until that time, continue working with the Platform.sh generated URLs.
+You need to:
```

Comment:
I'm not sure why the periods at the end should be avoided, since it's a bulleted list that's [neither a sequence nor options](https://developers.google.com/style/lists?hl=en#types-of-lists).

Reply:
Avoiding end punctuation isn't the goal. The goal is avoiding unnecessary repetition. Avoiding end punctuation is a side effect. The second exception in the section on [end punctuation in lists](https://developers.google.com/style/lists?hl=en#capitalization), which is the relevant section.